Build failure: Audiobookshelf

### Steps To Reproduce

Steps to reproduce the behavior:
1. build *audiobookshelf*

### Build log

```
error: builder for '/nix/store/n76x8hnpkpab4dl3z21bkis4p4kfwbzi-tone-0.1.5.drv' failed with exit code 1;
       last 10 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/ywgpqyqzzaxysmm5sfiwp6ad06y5sh16-source
       > source root is source
       > patching sources
       > configuring
       > Executing dotnetConfigureHook
       >   Determining projects to restore...
       > /build/source/tone/tone.csproj : error NU1102: Unable to find package Microsoft.NETCore.App.Crossgen2.linux-x64 with version (= 6.0.16)
       > /build/source/tone/tone.csproj : error NU1102:   - Found 1 version(s) in nugetSource [ Nearest version: 6.0.15 ]
       >   Failed to restore /build/source/tone/tone.csproj (in 2.82 sec).
       For full logs, run 'nix log /nix/store/n76x8hnpkpab4dl3z21bkis4p4kfwbzi-tone-0.1.5.drv'.
error: 1 dependencies of derivation '/nix/store/ms3jcxzdbmjri0kynm7rhkda8ayx4a5c-audiobookshelf-2.2.18.drv' failed to build

```

### Notify maintainers
@jvanbruegge
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

### Metadata

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 6.2.11-xanmod1, NixOS, 23.05 (Stoat), 23.05.20230430.da45bf6`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.15.0`
 - nixpkgs: `/etc/nix/flake-channels/nixpkgs`
```

--- a/pkgs/applications/audio/tone/default.nix
+++ b/pkgs/applications/audio/tone/default.nix
@@ -29,6 +29,6 @@ buildDotnetModule rec {
     description = "A cross platform utility to dump and modify audio metadata for a wide variety of formats";
     license = licenses.asl20;
     maintainers = [ maintainers.jvanbruegge ];
-    platforms = platforms.linux;
+    platforms = [ "x86_64-linux" ];
   };
 }

--- a/pkgs/applications/audio/tone/default.nix
+++ b/pkgs/applications/audio/tone/default.nix
@@ -26,7 +26,7 @@ buildDotnetModule rec {
 
   meta = with lib; {
     homepage = "https://github.com/sandreas/tone";
-    description = "tone is a cross platform utility to dump and modify audio metadata for a wide variety of formats";
+    description = "A cross platform utility to dump and modify audio metadata for a wide variety of formats";
     license = licenses.asl20;
     maintainers = [ maintainers.jvanbruegge ];
     platforms = platforms.linux;

--- a/pkgs/applications/audio/tone/default.nix
+++ b/pkgs/applications/audio/tone/default.nix
@@ -1,11 +1,6 @@
-{ lib, stdenv, fetchFromGitHub, fetchNuGet, linkFarmFromDrvs, buildDotnetModule, ffmpeg-full, msbuild, dotnetCorePackages }:
+{ lib, fetchFromGitHub, buildDotnetModule, ffmpeg-full, dotnetCorePackages }:
 
-let
-  nugetSource = linkFarmFromDrvs "nuget-packages" (
-    import ./nuget-deps.nix { inherit fetchNuGet; }
-  );
-
-in buildDotnetModule rec {
+buildDotnetModule rec {
   pname = "tone";
   version = "0.1.5";
 

--- a/pkgs/applications/audio/tone/nuget-deps.nix
+++ b/pkgs/applications/audio/tone/nuget-deps.nix
@@ -28,7 +28,7 @@
   (fetchNuGet { pname = "Microsoft.Extensions.Options.ConfigurationExtensions"; version = "7.0.0"; sha256 = "1liyprh0zha2vgmqh92n8kkjz61zwhr7g16f0gmr297z2rg1j5pj"; })
   (fetchNuGet { pname = "Microsoft.Extensions.Primitives"; version = "2.0.0"; sha256 = "1xppr5jbny04slyjgngxjdm0maxdh47vq481ps944d7jrfs0p3mb"; })
   (fetchNuGet { pname = "Microsoft.Extensions.Primitives"; version = "7.0.0"; sha256 = "1b4km9fszid9vp2zb3gya5ni9fn8bq62bzaas2ck2r7gs0sdys80"; })
-  (fetchNuGet { pname = "Microsoft.NETCore.App.Crossgen2.linux-x64"; version = "6.0.15"; sha256 = "0lcz7dniv3arkdzlmjgr9168rjb0an9xf3v3m3pdwjmy8yaipfba"; })
+  (fetchNuGet { pname = "Microsoft.NETCore.App.Crossgen2.linux-x64"; version = "6.0.16"; sha256 = "0rn46d2lrdyv8cjl2xxjj8zgahv1x4mz4z56vvw6zqfa10g5cnjh"; })
   (fetchNuGet { pname = "Newtonsoft.Json"; version = "13.0.2"; sha256 = "1p9splg1min274dpz7xdfgzrwkyfd3xlkygwpr1xgjvvyjvs6b0i"; })
   (fetchNuGet { pname = "Sandreas.AudioMetadata"; version = "0.1.1"; sha256 = "11ibv23h7qj5qshibmlsqmjca51dqbhib9p1gz66c5kqhk7ci38j"; })
   (fetchNuGet { pname = "Sandreas.Files"; version = "1.1.2"; sha256 = "08qk229q2y1dpdxdnp8xi9mgk8fgpjxrxm4z6ak8n09npp67nhn0"; })

